src/main/java/com/artipie/repo/RepoLayout.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.repo;
+
+import com.artipie.http.Slice;
+
+/**
+ * Repositories layout.
+ * <p>
+ * This object resolves repository HTTP endpoints for specified URL.
+ * Artipie supports two primary layout configurations:
+ * <ul>
+ *     <li>Flat layout - Artipie doesn't use any structural layout, all repositories
+ *     are located at the root of base path, e.g. URI {@code https://central.artipie.com/maven}
+ *     accesses {@code maven} repository.</li>
+ *     <li>User based - Artipie installation has multiple namespaces/organizations/users,
```

Comment:
`Org layout` and `User based` are different terms. I'd suggest sticking with only one, so the Javadoc would be consistent with the actual code